src/Microsoft.HttpRepl.Tests/OpenApi/EndpointMetadataReaderTests.cs

```diff
@@ -23,5 +26,28 @@ public void Read_WithJObjectFormatNotSupportedByAnyExistingReader_ReturnsNull()
 
             Assert.Null(endpointMetadataReader.Read(jobject));
         }
+
+        [Fact]
+        public void RegisterReader_AddNewReader_VerifyReadReturnsEndpointMetadataCollection()
+        {
+            string json = @"{
+  ""version"": ""1.0.0"",
```

Comment:
Instead of using "version" (which is kind of agnostic), what about using "fakeApi" so that it's more clear that this would only apply to the fake reader?

Reply:
fakeApi sounds better. Thanks!

Updated here:
https://github.com/aspnet/HttpRepl/commit/656236743ed10c9b14dd1b04eecfec61cb1b53af